Adaptors/MongoDB/src/TaskTable.cs

```diff
@@ -296,8 +296,7 @@ public class TaskTable : ITaskTable
                                   .ToListAsync(cancellationToken)
                                   .ConfigureAwait(false);
 
-    return res.Select(tuple => new TaskStatusCount(tuple.Status,
-                                                   tuple.Count));
+    return res;
```

Comment:
why is the transformation any more necessary?

Reply:
because it was recreating TaskStatusCount from TaskStatusCount already created in the mongodb request